### DISCUSSION TITLE
suggest: upload to Debian

### DISCUSSION THREAD

**Discussion by nbehrnd** [cyanfish/naps2#293] [General]
NAPS2 is developed with an open source code and libraries it uses carry either a GPL or MIT license.  Among the releases (already) are .deb for a seamless installation e.g., in Linux Debian.

Given this work, I wonder if there is there a particular reason why this helpful gem isn't yet in Debian's repositories.  License wise, the project already is compatible.  It equally would increase the project's visibility.

**Comment by cyanfish** [comment]
In theory I'd like to, but across Windows/Mac/Linux there are a lot of [different](https://github.com/cyanfish/naps2/issues?q=is%3Aissue+is%3Aopen+label%3Apackaging) distribution methods to potentially set up, and my priorities have been elsewhere lately.

**Comment by nbehrnd** [comment]
That's ok, too.  With a RFP ticket (request for packaging) to Debian I can
petition someone with the necessary experience in C# (not me) works on it.

**Comment by nbehrnd** [comment]
The RFP ticket to Debian is [here](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1062070).  Let's see if and when repology eventually can add the corresponding entry e.g., to the general survey

[![Packaging status](https://repology.org/badge/vertical-allrepos/naps2.svg)](https://repology.org/project/naps2/versions)

or the badge excluding unsupported OS releases:

[![Packaging status](https://repology.org/badge/vertical-allrepos/naps2.svg?exclude_unsupported=1&&columns=3)](https://repology.org/project/naps2/versions)
